setup.cfg

```diff
@@ -0,0 +1,7 @@
+[tox:tox]
+envlist = {py27,py34,py35,py36,py37,pypy,pypy3}
+skipsdist=True
```

Comment:
Lack of spaces around `=` looks weird here, considering they're everywhere else.